lib/services/consumptionManagement/lib/consumptionManagementClient.d.ts

```diff
@@ -40,14 +45,18 @@ declare class ConsumptionManagementClient extends AzureServiceClient {
    * @param {boolean} [options.generateClientRequestId] - When set to true a unique x-ms-client-request-id value is generated and included in each request. Default is true.
    *
    */
-  constructor(credentials: ServiceClientCredentials, subscriptionId: string, baseUri?: string, options?: AzureServiceClientOptions);
+  constructor(credentials: ServiceClientCredentials, subscriptionId: string, billingAccountId: string, grain: string, baseUri?: string, options?: AzureServiceClientOptions);
```

Comment:
breaking change. Why should `billingAccountId: string, grain: string` be properties of a management client.